CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 Records breaking changes from major version bumps
 
+## 50.0.0
+
+PR [#542](https://github.com/alphagov/digitalmarketplace-scripts/pull/542)
+
+Bumping the werkzeug and itsdangerous version restrictions. Test & deploy with extra care.
```

Comment:
Ooo are we going to update the pinned `itsdangerous==0.24` on every app then? Fun!

Reply:
Oyes - I don't think there's any point in doing the werkzeug bump without it.

Reply:
... I am glad this is a major bump.

Reply:
OK, looking through the `werkzeug` changelog I don't think we need the `itsdangerous` bump yet. However there are breaking changes to how `werkzeug` imports are used: https://werkzeug.palletsprojects.com/en/0.16.x/changes/

I think we're mostly OK but the AV API does a direct import, which will need changing.

Reply:
> but the AV API does a direct import

I actually think what's being used in that case (`test_sns.py`, right?) doesn't count as a "top level import" because it's actually accessing a submodule of the top-level. "top level" I think means the uncategorized melange of objects which packages will often put at the top level for convenience.